src/apps/components/pages/ClassPropertiesTab.svelte

```diff
@@ -332,6 +332,15 @@
             }
         }
 
+        &__field--input {
+            border: 1px solid #7a7971;
```

Comment:
If this color is available as a variable replace the hex code with the variable.

Reply:
This isn't defined anywhere that I could find.